src/__mocks__/gatsby.js

```diff
@@ -0,0 +1,27 @@
+const React = require('react');
+
+const gatsby = jest.requireActual('gatsby');
+
+module.exports = {
+  ...gatsby,
+  graphql: jest.fn(),
+  Link: jest.fn().mockImplementation((
+    // these props are invalid for an `a` tag
+    ({
+      activeClassName,
```

Comment:
Like the question you asked earlier about this file, I don't think we need it anymore since we're no longer using StaticQuery or Link components in the code base.

Reply:
I'm going to leave it for now. We can remove it later if we need to.